trams_api_spec/trams_api_spec.yaml

```diff
@@ -834,6 +854,36 @@ definitions:
         type: "string"
       status:
         type: "string"
+  AcademyTransferProjectSearch:
```

Comment:
Maybe we rename this to `ListItem` or similar, search result is maybe fine if its an implicit pattern that we return a list friendly truncated object, but maybe we should just be clear

Reply:
Yeah I was struggling for a name - ListItem sounds nice